src/route.rs

```diff
@@ -2,7 +2,7 @@ use crate::{router::Router, Endpoint};
 
 /// A handle to a route.
 ///
-/// All HTTP requests are made against resources. After using [`App::at`] (or
+/// All HTTP requests are made against resources. After using [`App::at`](crate::app::App::new) (or
 /// [`Route::at`]) to establish a route, the `Route` type can be used to
 /// establish endpoints for various HTTP methods at that path. Also, using
 /// `nest`, it can be used to set up a subrouter.
```

Comment:
```suggestion
 /// `nest`, it can be used to set up a subrouter.

[`App::at`]: ../struct.App.html#method.at
```

Reply:
Why should I change the notation of the path?

Reply:
My understanding was that [`tide::Route`](https://docs.rs/tide/0.2.0/tide/struct.Route.html) wasn't rendering the link to `App::at` correctly.

Using link blocks at the bottom of a paragraph is pretty standard practice in Rust's stdlib, so I that's why I proposed making this change.

Reply:
Ok.  I did not know that I could include a link block. Thanks 😄